src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -59,6 +58,17 @@ ov::Tensor create_and_fill_tensor(const ov::element::Type element_type,
     return tensor;
 }
 
+ov::Tensor create_and_fill_tensor(const ov::element::Type element_type,
```

Comment:
mark it as `todo`  to remove the function in the future

Reply:
done